Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.6                                                                                                                                                                                         
+                                                                                 
+RUN apt-get update \                                                             
+    && apt-get install -y --no-install-recommends \                              
+        postgresql-client \                                                      
```

Comment:
Just curious, why are we installing the PostgreSQL client package?

Reply:
> **@nlMeminger**:
> The dangers of finding guides on the internet and copying and pasting.
> Realistically it shouldn’t be there, but it doesn’t hurt right now.

Okay, cool. I wanted to double-check to be sure. In this case, you can drop the `apt-get install` step completely to reduce container build time.